crypto-logging-mapper/src/test/kotlin/MapperTest.kt

```diff
@@ -0,0 +1,47 @@
+import context.CryptoOrderContext
+import kotlinx.datetime.Instant
+import models.*
+import models.commands.CryptoOrderCommands
+import org.junit.Test
+import kotlin.test.assertEquals
+
+class MapperTest {
+
+    @Test
+    fun fromContext() {
+        val context = CryptoOrderContext(
+            requestId = CryptoRequestId("1234"),
+            command = CryptoOrderCommands.CREATE,
+            ordersResponse = mutableListOf(
+                CryptoOrder(
+                    ownerId = CryptoUserId("234"),
+                    orderId = CryptoOrderId("order-id"),
+                    created = Instant.fromEpochMilliseconds(1231313131),
+                    orderState = CryptoOrderState.ACTIVE,
+                    amount = 3.3.toBigDecimal(),
```

Comment:
```suggestion
                    amount = "3.3".toBigDecimal(),
```
double - приближенный тип. Для 3.3 нет точного значения в double. После преобразования в биг десимал будут сюрпризы в виде девяток или много нулей и 1 на конце

Reply:
верно. извиняюсь, опечатался